EIPS/eip-2070.md

```diff
@@ -22,38 +22,40 @@ This meta-EIP specifies the changes included in the Ethereum hardfork named Berl
 
 - TBD
 
-### Accepted EIPs
+### EFI accepted EIPs
```

Comment:
I think we should just use Accepted here, as we did in the call.

Reply:
Okay, sure.